WcaOnRails/app/views/static_pages/robots.txt.erb

```diff
@@ -3,3 +3,7 @@
 User-agent: *
 Disallow: /
 <% end %>
+
+User-agent: *
```

Comment:
How do you feel about adding a comment here with a link to https://docs.google.com/document/d/1XND2oUwGZbwXLXaDMDY4moQwnsEpET7zSaqTukrULUM/ ?